### PR TITLE
Created test for two grades and implemented

### PR DESCRIPTION
**Pull request by Manny-Edwards**

### CODE REVIEW

File: test_grades.py [hunk 1]
```diff
@@ -10,3 +10,8 @@ def test_zero_grades():
 def test_single_grade():
     grades = [42]
     assert compute_hw_average(grades) == 42
+
+
+def test_two_grades():
+    grades = [2,4]
```
Comment (Charles-Peeke) [inline test_grades.py]: I think that there should be a test to test for floating point numbers

Comment (Manny-Edwards) [reply test_grades.py]: My most recent commit changed it

File: test_grades.py [hunk 2]
```diff
@@ -13,5 +13,5 @@ def test_single_grade():
 
 
 def test_two_grades():
-    grades = [2,4]
-    assert compute_hw_average(grades) == 3
+    grades = [2.0,4.0]
+    assert compute_hw_average(grades) == 3.0
```
Comment (Charles-Peeke) [inline test_grades.py]: I think that this test should use grades where the answer is not an integer, but rather a floating point number that is not rounded.

Comment (Manny-Edwards) [reply test_grades.py]: Problem was resolved. The test is now floart friendlty

Comment (Manny-Edwards) [reply test_grades.py]: *float friendly

**Pull request merged**